serverless.yml

```diff
@@ -101,14 +101,20 @@ functions:
 
   #############################################################################
   # OPTION(canary): Canary-deployed Lambda using serverless-plugin-canary-deployments
-  ############################################################################
+  # To watch the canary deploy's progress for the sandbox environment, view its
+  # CodeDeploy project here:
+  # https://us-east-1.console.aws.amazon.com/codesuite/codedeploy/applications/sls-simple-reference-sandbox-SlssimplereferencesandboxDeploymentApplication-LI2MS1UTYKZT?region=us-east-1
```

Comment:
How does one determine this URL? Like the `LI2MS1UTYKZT` part?

Maybe add a starter URL and then what to look for and click?